julietezekwe/iReporter-API
# #168237485 Setup JWT for token-based authentication & authorization

#### What does this PR do?
This PR ensures user authentication and authorization before API requests. 

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
- Checkout to this branch
- run `bundle install` in terminal
- run `rspec`
- Notice that all the `AuthorizeApiRequests`
 
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#168237485](https://www.pivotaltracker.com/story/show/168237485)
